tests/intagration_tests.py

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env python
+"""
+Authors: Fred Buchanan
+License: Apache 2.0
+Description: Intgation tests for coconut
```

Comment:
The proper spelling is `integration`. Can you change both the docstring and the file name?